templates/frontend/pages/userSubscriptions.tpl

```diff
@@ -181,34 +178,34 @@
 											{assign var="subscriptionStatus" value=$userInstitutionalSubscription->getStatus()}
 											{assign var="isNonExpiring" value=$userInstitutionalSubscription->isNonExpiring()}
 											{if $paymentsEnabled && $subscriptionStatus == $smarty.const.SUBSCRIPTION_STATUS_AWAITING_ONLINE_PAYMENT}
-												<span class="subscription_disabled">
+												<span>
```

Comment:
subscription_disabled class here and downwards that is deleted in this branch is used to discriminate situation when the option is disabled; it's for using another styling than for general elements, do you have other ideas how we can manage such situations?

Reply:
I see! I will create new styles for it, keep it removed for now. I’ve noted it, and will make a separate PR to complete these styles.